src/main/java/org/spongepowered/api/profile/GameProfileCache.java

```diff
@@ -75,15 +75,15 @@
      * @return The profile, if present, or {@link Optional#empty()} if
      *     the cache did not contain a profile with the provided id
      */
-    Optional<GameProfile> byId(UUID uniqueId);
+    Optional<GameProfile> findById(UUID uniqueId);
```

Comment:
I originally wanted to drop "byId" etc. but the problem is that findByIds and findByNames have the same erasure so wouldn't work out so well.

We could have `find` for the singular returns, and keep the current names for the... well, current values!